src/Control/Effect/Choose.hs-boot

```diff
@@ -0,0 +1,10 @@
+module Control.Effect.Choose
+( Choose(..)
+) where
+
+import Control.Effect.Class
+
+newtype Choose m k
+  = Choose (Bool -> m k)
+
+instance HFunctor Choose
```

Comment:
The `.hs-boot` files for the effects have to export the effect datatypes, but also the `HFunctor` instance, since that’s a superclass of `Carrier`. We don’t need the `Effect` or `Functor` instances, since they’re not involved in the `base` or `transformers` instances; and certainly not the `Generic1` instances.